swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/TunnelManager.swift

```diff
@@ -279,7 +279,7 @@ public class TunnelManager {
       try session().sendProviderMessage(encoder.encode(TunnelMessage.getResourceList(resourceListHash))) { data in
         if let data = data {
           self.resourceListHash = Data(SHA256.hash(data: data))
-          var decoder = JSONDecoder()
+          let decoder = JSONDecoder()
```

Comment:
Good catch, I thought changing the `keyDecodingStrategy` would require this to be mutable